### PR TITLE
it seems a few more scripts were added

### PR DESCRIPTION
**Pull request by SolarAquarion**

### CODE REVIEW

File: archlinuxcn/llvm-git/PKGBUILD [hunk 1]
```diff
@@ -217,6 +217,9 @@ package_clang-git() {
 
     # Move analyzer scripts out of /usr/libexec
     mv "$pkgdir"/usr/libexec/{ccc,c++}-analyzer "$pkgdir/usr/lib/clang/"
+    mv "$pkgdir"/usr/libexec/analyze-{c++,cc} "$pkgdir/usr/lib/clang/"
+    mv "$pkgdir"/usr/libexec/intercept-{c++,cc} "$pkgdir/usr/lib/clang/"
+
     rmdir "$pkgdir/usr/libexec"
```
Comment (yuyichao) [inline archlinuxcn/llvm-git/PKGBUILD:223]: Given this, should this just move everything under `libexec` over?

Comment (SolarAquarion) [reply archlinuxcn/llvm-git/PKGBUILD:223]: Yeah, i guess?

File: archlinuxcn/llvm-git/PKGBUILD [hunk 2]
```diff
@@ -99,7 +99,7 @@ build() {
         -D LLVM_ENABLE_PROJECTS="lldb;polly;compiler-rt;lld;clang-tools-extra;clang" \
         -D TENSORFLOW_C_LIB_PATH="/usr/" # Force on
 
-    ninja -C _build LLVMgold all ocaml_doc
+    ninja -j8 -C _build LLVMgold all ocaml_doc
```
Comment (yuyichao) [inline archlinuxcn/llvm-git/PKGBUILD:102]: And does it run out of memory or something without this on the buildbot?

File: archlinuxcn/llvm-git/PKGBUILD [hunk 3]
```diff
@@ -217,6 +217,9 @@ package_clang-git() {
 
     # Move analyzer scripts out of /usr/libexec
     mv "$pkgdir"/usr/libexec/{ccc,c++}-analyzer "$pkgdir/usr/lib/clang/"
+    mv "$pkgdir"/usr/libexec/analyze-{c++,cc} "$pkgdir/usr/lib/clang/"
+    mv "$pkgdir"/usr/libexec/intercept-{c++,cc} "$pkgdir/usr/lib/clang/"
+
     rmdir "$pkgdir/usr/libexec"
     sed -i 's|libexec|lib/clang|' "$pkgdir/usr/bin/scan-build"
```
Comment (yuyichao) [inline archlinuxcn/llvm-git/PKGBUILD:224]: And if you have a locally compiled version, you might want to check if there's any new references to libexec that needs to be patched up now.

File: archlinuxcn/llvm-git/PKGBUILD (hunk 2 above)
Comment (SolarAquarion) [reply archlinuxcn/llvm-git/PKGBUILD:102]: no, it's a local thing that i made a bit of a mistake on

File: archlinuxcn/llvm-git/PKGBUILD (hunk 3 above)
Comment (SolarAquarion) [reply archlinuxcn/llvm-git/PKGBUILD:224]: nothing new

Comment (yuyichao) [reply archlinuxcn/llvm-git/PKGBUILD:224]: ok

**Pull request closed without merging**